main.go

```diff
@@ -79,5 +128,23 @@ func main() {
 		log.Printf("serving proxy at port %v\n", *port)
 	}
 
-	log.Fatal(http.ListenAndServe(":"+*port, proxy))
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+
+        if isBot(r, bots) {
+            // pass the request to the reverse proxy
+			var userAgent= r.UserAgent()
+			requestDump, err := httputil.DumpRequest(r, true)
+			if err != nil {
+  				fmt.Println(err)
+			}
+			message := fmt.Sprintf("Ignored request with userAgent %s and full request %s", userAgent, string(requestDump))
+			log.Println(message)
+			w.WriteHeader(http.StatusOK)
+			return
+            
+        } else {
+			proxy.ServeHTTP(w, r)
```

Comment:
Should the else part just fall through to http.ListenAndServe like it did before?

Reply:
maybe :) Not super familiar with these functions - but the current approach seems to work. If I put the `ListenAndServe` from row 149 inside the else it doenst :(